specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/CheckManagementGroupNameAvailability.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2020-05-01",
+    "checkNameAvailabilityRequest": {
+      "name": "nameTocheck",
+      "type": "Microsoft.Management/managementGroup"
```

Comment:
It should be "Microsoft.Management/managementGroups" type

Reply:
"CheckNameAvailabilityRequest" definition was pointing to  "Microsoft.Management/managementGroup", and the validation was throwing an error.
Will make the change in the definition and revert this change in next iteration